eng/Generate.ps1

```diff
@@ -30,7 +30,13 @@ function Invoke-AutoRest($baseOutput, $title, $autoRestArguments)
 
     if ($clean)
     {
-        Get-ChildItem $outputPath -Filter Generated -Directory -Recurse | Remove-Item -Force -Recurse;
+        try
```

Comment:
This has never failed for me is some file locked?

Reply:
Removed the changes.